### DISCUSSION TITLE
issue with divs

### DISCUSSION THREAD

**Discussion by TheAlexBarth** [quarto-dev/quarto-cli#988] [Q&A] [answered]
When specifying classes on a div, it doesn't render correctly from rmarkdown:

for example, using 
:::{.content-visible unless-format='pdf'} 

<iframe src='somewebcontent'>

:::

The pdf rendering will have:

:::{.content-visible unless-format='pdf'} 
:::

**Comment by cscheid** [comment, marked as answer]
The issue here is that the specific HTML syntax is confusing pandoc. Try closing the iframe tag, either by `<iframe src='somewebcontent'/>` or by `<iframe src='somewebcontent'/></iframe>`

**Reply by TheAlexBarth** [reply]
Thanks!
